libs/vue/src/schematics/application/schematic.ts

```diff
@@ -108,9 +108,10 @@ function getEslintConfig(options: NormalizedSchema) {
   const eslintConfig: any = {
     extends: [
       `plugin:vue/${options.isVue3 ? 'vue3-' : ''}essential`,
+      'eslint:recommended',
```

Comment:
[Nx already includes this.](https://github.com/nrwl/nx/blob/11.5.1/packages/eslint-plugin-nx/src/configs/typescript.ts#L24) Order does matter, so this isn't equivalent to the resulting configuration if this wasn't here, but I don't think it should be changed as a part of this PR if at all.